tracpro/polls/tests/test_forms.py

```diff
@@ -78,3 +79,25 @@ def test_numeric_invalid(self):
             'numeric': ['Select a valid choice. '
                         'invalid is not one of the available choices.'],
         })
+
+
+class TestActivePollsForm(TracProTest):
```

Comment:
I have added a test for the task specifically. Let me know if you have any more suggestions! Thanks.